src/converters/app-converter.ts

```diff
@@ -0,0 +1,388 @@
+            }
+        }
+
+        // Register the outputs as outputs of the component resource.
+        for (const [outputId, args] of Object.entries(this.stack.outputs ?? {})) {
+            this.host.registerOutput(outputId, this.processIntrinsics(args.Value));
+        }
+
+        for (let i = dependencyGraphNodes.length - 1; i >= 0; i--) {
+            const n = dependencyGraphNodes[i];
+            if (!n.resource) {
+                (<CdkConstruct>this.constructs.get(n.construct)!).done();
+            }
+        }
+    }
+
+    private stackDependsOn(dependencies: Set<ArtifactConverter>): pulumi.Resource[] {
+        const dependsOn: pulumi.Resource[] = [];
+        for (const d of dependencies) {
+            if (d instanceof FileAssetManifestConverter) {
+                this.resources.set(d.id, { resource: d.file, resourceType: d.resourceType });
+                dependsOn.push(d.file);
+            }
+            // TODO: handle docker images
+        }
+        return dependsOn;
+    }
+
+    private mapParameter(logicalId: string, typeName: string, defaultValue: any | undefined) {
+        // TODO: support arbitrary parameters?
+
+        if (!typeName.startsWith('AWS::SSM::Parameter::')) {
+            throw new Error(`unsupported parameter ${logicalId} of type ${typeName}`);
+        }
+        if (defaultValue === undefined) {
+            throw new Error(`unsupported parameter ${logicalId} with no default value`);
+        }
+
+        function parameterValue(parent: pulumi.Resource): any {
+            const key = defaultValue;
+            const paramType = typeName.slice('AWS::SSM::Parameter::'.length);
+            if (paramType.startsWith('Value<')) {
+                const type = paramType.slice('Value<'.length);
+                if (type.startsWith('List<') || type === 'CommaDelimitedList>') {
+                    return getSsmParameterList({ name: key }, { parent }).then((v) => v.value);
+                }
+                return getSsmParameterString({ name: key }, { parent }).then((v) => v.value);
+            }
+            return key;
+        }
+
+        this.parameters.set(logicalId, parameterValue(this.host));
+    }
+
+    private mapResource(
+        logicalId: string,
+        typeName: string,
+        props: any,
+        options: pulumi.ResourceOptions,
+    ): ResourceMapping {
+        if (this.host.options?.remapCloudControlResource !== undefined) {
+            const res = this.host.options.remapCloudControlResource(logicalId, typeName, props, options);
+            if (res !== undefined) {
+                debug(`remapped ${logicalId}`);
+                return res;
+            }
+        }
+
+        const awsMapping = mapToAwsResource(logicalId, typeName, props, options);
+        if (awsMapping !== undefined) {
+            debug(`mapped ${logicalId} to classic AWS resource(s)`);
+            return awsMapping;
+        }
+
+        return mapToCfnResource(logicalId, typeName, props, options);
```

Comment:
should we debug log here as well? And an additional one in case we didn't find a mapping?